gogoogledm.go

```diff
@@ -9,6 +9,8 @@ import (
 	"net/url"
 	"strings"
 	"time"
+
+	"golang.org/x/net/context"
```

Comment:
I think it should be better to use "context" here. But no strong opinion.

@arkan WDYT?

Reply:
Same for me, not having a strong opinion on that point and can change if we prefer to have "context".